lib/rules/method.js

```diff
@@ -120,6 +120,7 @@ function checkCallExpression(ruleHelper, callExpr, node) {
     case "Super":
     case "CallExpression":
     case "ThisExpression":
+    case "TaggedTemplateExpression":
```

Comment:
Putting it here will make the linter skip these expressions. That doesn't seem to be what we want.